### PR TITLE
Added query app and urls.py in server/urls.py

### PR DESCRIPTION
**Pull request by sahil9001**
Fixes issue #10 
Description
This commit adds new app named 'query'.

Type of Change: (Delete irrelevant options)
Added query app

Checklist:

- [x] My PR follows the style guidelines of this project

- [x] I have performed a self-review of my own code or materials

Code/Quality Assurance

- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by naveennvrgup** [comment]
@sahil9001 We like your optimism. One PR should be targeted at one issue only! Please create separate PR for issues #9 and #10.

**Comment by sahil9001** [comment]
@naveennvrgup

File: server/urls.py [hunk 1]
```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include 
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('api/query',include('query.urls'))
```
Comment (naveennvrgup) [inline server/urls.py]:
```suggestion
    path('api/query/',include('query.urls'))
```

**Review by naveennvrgup — COMMENTED** [review]
Thanks for the code. Well done :fire:. Please do the requested changes. Then ill merge it.

**Comment by naveennvrgup** [comment]
Also update the PR description. Ill provide you the format in the slack for now.

**Comment by sahil9001** [comment]
@naveennvrgup Edited the description. Is it fine now?

**Review by naveennvrgup — APPROVED** [review]
Nice job on the first PR! I am merging this PR now. In your next PR i expect:
1. proper commit msg
1. dont delete the formating in the PR template
1. dont PR from dev branch
1. and other things I have mentioned earlier.

**Congrats on your first PR :champagne: :firecracker: :tada: **

**Pull request merged**